config/nrfconnect/chip-module/Kconfig

```diff
@@ -202,6 +202,54 @@ config CHIP_FACTORY_RESET_ERASE_NVS
 	bool
 	default y if CHIP_FACTORY_DATA || CHIP_FACTORY_DATA_CUSTOM_BACKEND
 
+config CHIP_WIFI_CONNECTION_RECOVERY_ENABLE
```

Comment:
We usually don't use `ENABLE` in the Kconfig name, as its boolean type means it can be enabled/disabled by setting y/n.

Reply:
Btw. I was wondering if we would like to enable it using Kconfig. It looks for me like a functionality that we would like to have always enabled, so maybe there is no need to allow disabling it. But let it be, we can remove this kconfig someday, once we will be sure it's working well.